src/App.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
-import { BrowserRouter as Router, Switch, Route, Link} from 'react-router-dom';
-import Routes from './routes'
+import { BrowserRouter as Router } from 'react-router-dom';
+import Routes from './routes';
+import NavBar from './Components/Layouts/Navbar';
+
 
 function App() {
   return (
     <div className="App">
       <Router>
+        <NavBar/>
```

Comment:
The NavBar goes outside the router.
Reason: Each time you you to another route, the navbar will be rerendered again. That shouldn't be the case since it's the same for all routes.